src/queries/auth.ts

```diff
@@ -8,12 +8,15 @@ import NavigationService from 'services/NavigationService';
 export const useLoginMutation = () => {
   const { refetch } = useContext(UserContext);
 
-  return useMutation<object, Error, { username: string; password: string }>(authService.login, {
-    retry: false,
-    onSuccess: async () => {
-      refetch();
-    },
-  });
+  return useMutation<object, Error, { username: string; password: string }>(
```

Comment:
I know it was already like this, but it's really bad having a return value type `object`. We should always use custom interfaces with expected props.
Not sure how many places we need to cover. 
This can be a separate ticket.